recirq/otoc/loschmidt/tilted_square_lattice/gen-small-v1.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+import cirq
+from recirq.otoc.loschmidt.tilted_square_lattice import \
+    get_all_tilted_square_lattice_executables
+
+FN = 'loschmidt.tilted_square_lattice.small-v1.json.gz'
+
+
+def main():
+    exes = get_all_tilted_square_lattice_executables(
+        min_side_length=2, max_side_length=3, side_length_step=1,
+        n_instances=3,
+        macrocycle_depths=np.arange(0, 4 + 1, 1))
+    print(len(exes), 'executables')
+
+    cirq.to_json_gzip(exes, FN)
```

Comment:
Should the output filename be a flag?

Reply:
The current design is to have a 1:1 correspondence between these generation scripts and the serialized `QuantumExecutableGroup`. I consider this file to be a configuration file like a `yaml` or `textproto` file. I could just use a yaml file or something; but I find often times the schema of configuration files gets extensions bolted on to it to handle new ways of configuring things so here I'm just using a real scripting language from the start.

Remember: we need to carefully control the benchmarks we run for accurate comparison over time. If you want to run an ad-hoc benchmark with your own parameter choices, you should likely create a new script to do what you want.